client/src/parsec/login.ts

```diff
@@ -396,22 +396,27 @@ export async function getCurrentAvailableDevice(): Promise<Result<AvailableDevic
   }
 }
 
-export async function changeAuthentication(
+export async function updateDeviceChangeAuthentication(
   accessStrategy: DeviceAccessStrategy,
   saveStrategy: DeviceSaveStrategy,
-): Promise<Result<null, ClientChangeAuthenticationError>> {
+): Promise<Result<AvailableDevice, UpdateDeviceError>> {
   if (!needsMocks()) {
     const clientConfig = getClientConfig();
-    return await libparsec.clientChangeAuthentication(clientConfig, accessStrategy, saveStrategy);
+    return await libparsec.updateDeviceChangeAuthentication(clientConfig.configDir, accessStrategy, saveStrategy);
   } else {
     // Fake an error
     if (
       accessStrategy.tag === DeviceAccessStrategyTag.Password &&
       (accessStrategy as DeviceAccessStrategyPassword).password !== 'P@ssw0rd.'
     ) {
-      return { ok: false, error: { tag: ClientChangeAuthenticationErrorTag.DecryptionFailed, error: 'Invalid password' } };
+      return { ok: false, error: { tag: UpdateDeviceErrorTag.DecryptionFailed, error: 'Invalid password' } };
     }
-    return { ok: true, value: null };
+    return {
+      ok: true,
+      // @ts-ignore: Too cumbersome to create a full `AvailableDevice` object,
```

Comment:
You can just c/c from https://github.com/Scille/parsec-cloud/blob/050fceb7664512f9f1f1ca863a165f9a999711a6/client/src/parsec/claim_device.ts#L253 It may be easier than trying to trick the linter.
This code will be gone very soon anyway.

Reply:
done, thanks :pray: